src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.cs

```diff
@@ -39,7 +43,7 @@ private static int Main(string[] args)
             Type t = null;
             MethodInfo mi = null;
             object instance = null;
-            int exitCode;
+            int exitCode = int.MaxValue;
```

Comment:
This was necessary because the compiler can't detect that the catch block always throws?